pandas/tests/groupby/test_categorical.py

```diff
@@ -1235,10 +1289,10 @@ def df_cat(df):
 @pytest.mark.parametrize("operation", ["agg", "apply"])
 def test_seriesgroupby_observed_true(df_cat, operation):
     # GH 24880
```

Comment:
Nit: Could you add the GH reference related to why this test changed?

Reply:
Done.